src/main/java/duke/command/NoopCommand.java

```diff
@@ -0,0 +1,27 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * Represents a no operation command.
+ */
+public class NoopCommand extends Command {
```

Comment:
Could consider InvalidCommand instead of NoopCommand as a more intuitive name here.